code/MANAGER/MANAGER.py

```diff
@@ -34,8 +34,8 @@ def __init__(self) -> None:
         self.__epc_config: STM_FLASH_Epc_Conf_c = None
         self.__status: TERM_Option_e = TERM_Option_e.CONF_DEV
         self.__power = PWR_c(config_path_file = f"") #TODO: check path
-    
-    
+        self.__stm: STM_FLASH_c = None  #TODO: check path

```

Comment:
Create an default EPC_CONF and pass it to STM_Flash

Reply:
Done.